lib/fastlane/plugin/wpmreleasetoolkit/helper/github_helper.rb

```diff
@@ -70,7 +70,7 @@ def self.get_last_milestone(repository)
           else
             begin
               last_vcomps = last_stone[:title].split[0].split('.')
-              last_stone = mile if mile_vcomps[0] > last_vcomps[0] || mile_vcomps[1] > last_vcomps[1]
+              last_stone = mile if Integer(mile_vcomps[0]) > Integer(last_vcomps[0]) || Integer(mile_vcomps[1]) > Integer(last_vcomps[1])
```

Comment:
Note that `Integer(x)` will _crash_ if `x` cannot be interpreted as an Integer — as opposed to `x.to_i` which will return `0` in such cases.

I'm not sure what's best and what we want, but in the (unlikely, but still possible) case that we have milestones like `alpha.1` in GitHub that will end up having `mile_vcomps[0]` be `'alpha'` and have this crash, I wonder if `.to_i` wouldn't be better? (That being said, we have a `rescue StandardError` block for if `Integer(x)` fails, so… 🤷 )

Reply:
I'd prefer a crash over unpredictable behaviour every time. If we have a special case, we should handle that and a crash is a great reminder for it.

Reply:
Good point.